### ISSUE TITLE
Update ClamAV

### ISSUE THREAD

**Issue by gotjoshua** [hardware/mailserver#349]
#### Classification

* [X] Feature/Enhancement

#### Reproducibility

* [X] Always

```
mailserver      | 2019-01-28T11:26:11.451554+00:00 mail freshclam[642]: Your ClamAV installation is OUTDATED!
mailserver      | 2019-01-28T11:26:11.451604+00:00 mail freshclam[642]: Local version: 0.100.2 Recommended version: 0.101.1
mailserver      | 2019-01-28T11:26:11.451634+00:00 mail freshclam[642]: DON'T PANIC! Read https://www.clamav.net/documents/upgrading-clamav
```

**Comment by denji** [comment]
https://tracker.debian.org/pkg/clamav
https://security-tracker.debian.org/tracker/source-package/clamav

**Comment by gotjoshua** [comment]
does that mean, its out of your hands and will update automatically when debian updates?

**Comment by denji** [comment]
@gotjoshuatr This only means that it will be required to keep the current version up to date and to batch it. ClamAV/Yara bugs #280

**Comment by denji** [comment]
Antivirus module (available from Rspamd 1.4) provides integration with virus scanners. Currently supported are:

* ClamAV
* F-Prot
* Sophos (via SAVDI)
* Avira (via SAVAPI)
* Kaspersky antivirus (from 1.8.3)

https://rspamd.com/doc/modules/antivirus.html

**Comment by hardware** [comment]
> does that mean, its out of your hands and will update automatically when debian updates?

Yes.
